04-onion-routing.md

```diff
@@ -263,6 +264,15 @@ It is formatted according to the Type-Length-Value format defined in [BOLT #1](0
     2. data:
         * [`32*byte`:`payment_secret`]
         * [`tu64`:`total_msat`]
+    1. type: 10 (`outgoing_node_id`)
+    2. data:
+        * [`point`:`outgoing_node_id`]
```

Comment:
This addition seems orthogonal to the trampoline proposal (it's not wrong; scid is just a shortcut for spelling out this outgoing_node_id anyway)?  Not sure it's necessary though.

Reply:
It is really necessary to point to the next trampoline node. Imagine Alice chooses two trampoline nodes T1 and T2. She needs to tell T1 to forward to T2. They may not be directly connected, and even if they were, Alice doesn't necessarily have the graph information to find a matching scid. The natural way to target T2 is to include its `node_id`. Do you think it should be done differently?